src/common/components/select-repository-list/index.js

```diff
@@ -136,7 +139,9 @@ export class SelectRepositoryListComponent extends Component {
   }
 
   renderPageLinks(pageLinks) {
-    if (pageLinks) {
+    const hasPagination = !!(pageLinks && (pageLinks.first || pageLinks.last));
```

Comment:
Correctly checking if pagination should be visible - only if we have any paging links.